lib/conjur/command/users.rb

```diff
@@ -47,7 +47,11 @@ class Conjur::Command::Users < Conjur::Command
           if api.username == options[:user]
             exit_now! 'To rotate the API key of the currently logged-in user, use this command without any flags or options'
           end
-          puts api.resource([ Conjur.configuration.account, "user", options[:user] ].join(":")).rotate_api_key
+          user_resource_id = [Conjur.configuration.account, "user", options[:user]].join(":")
+          unless api.resource(user_resource_id).exists?
+            exit_now! "user '#{options[:user]}' not found"
```

Comment:
@egvili it seems that other `exit_now!` messages start with a capital letter. Can you align this?

Reply:
I was aligning it to the same message coming from host rotate api. But I agree that other `exit_now!` messages begin with capital letter so I'll make that change. (Should I change also for host?)

Reply:
yes, please capitalize all the non-capital letter messages.